02 Algorithm Reference/02 Initializing Algorithms/02 Setting Cash/content.html

```diff
@@ -1,3 +1,4 @@
+<meta name="tag" content="using quantconnect" />
```

Comment:
this "Initializing Algorithms" wouldn't be using quantconnect -- its about the API

Reply:
I'm not sure If I'm just not seeing the tag, but I don't see any tag for "API", in the google docs. Do I just put the API tag in anyways ?

Reply:
I'm guessing just remove the tag I have presently and leave this particular html without tags ?

Reply:
Just reviewed tags; you're correct there was no tag to describe this section;
Added a new one: "Portfolio Setup" under securities and portfolio;

So you'd need 2 tags:
"portfolio setup" and "securities and portfolio"